eunkyung/app.js

```diff
@@ -34,25 +34,141 @@ app.get('/ping', (req, res) => {
 })
 
 app.post('/sign_up', async (req, res, next) => {
-    const { name, password, email } = req.body
+    const { name, password, email, profile_image } = req.body
```

Comment:
프론트엔드와 백엔드 모두 사용하는 언어가 동일할 때 발생하는 이점은 이처럼 언어 별 컨벤션을 서로 맞출 수 있다는 장점입니다.

JavaScript 언어는 기본적으로 camelCase를 사용하는 컨벤션이 일반적이기 때문에 관련 부분 모두 camelCase로 수정해주면 좋을 것 같습니다!
```suggestion
    const { name, password, email, profileImage } = req.body
```